bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -192,11 +207,16 @@ spec:
             --nb-address "tcp://${db_ip}:{{.OVN_NB_PORT}}" \
             --sb-address "tcp://${db_ip}:{{.OVN_SB_PORT}}" \
             --nodeport --gateway-mode local \
+            ${hybrid_overlay_flags} \
             --pidfile /var/run/openvswitch/ovnkube-node.pid \
-            --loglevel "${OVN_KUBE_LOG_LEVEL}" --logfile /dev/stdout
+            --loglevel 4  --logfile /dev/stdout
```

Comment:
@JacobTanenbaum need to revert this change to log level

Reply:
done